tests/requirements/reqs-base.txt

```diff
@@ -29,7 +29,7 @@ mock
 pytz
 ecs_logging
 structlog
-wrapt>=1.14.1,<1.15.0
+wrapt>=1.14.1,!=1.15.0
```

Comment:
I adapted this like in the setup.cfg. Personally I would actually go for wrapt >=1.16.0 here, but I don't know the preferences in the pinning workflow.

Reply:
This is one is fine